mediagrains_py36/numpy/__init__.py

```diff
@@ -0,0 +1,220 @@
+    elif fmt == CogFrameFormat.YUYV:
+        return [
+            as_strided(data,
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*2)).transpose(),
+            as_strided(data[1:],
+                       shape=(components[0].height, components[0].width//2),
+                       strides=(components[0].stride, data.itemsize*4)).transpose(),
+            as_strided(data[3:],
+                       shape=(components[0].height, components[0].width//2),
+                       strides=(components[0].stride, data.itemsize*4)).transpose()]
+    elif fmt == CogFrameFormat.RGB:
+        return [
+            as_strided(data,
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*3)).transpose(),
+            as_strided(data[1:],
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*3)).transpose(),
+            as_strided(data[2:],
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*3)).transpose()]
+    elif fmt in [CogFrameFormat.RGBx,
+                 CogFrameFormat.RGBA,
+                 CogFrameFormat.BGRx,
+                 CogFrameFormat.BGRx]:
+        return [
+            as_strided(data,
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*4)).transpose(),
+            as_strided(data[1:],
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*4)).transpose(),
+            as_strided(data[2:],
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*4)).transpose()]
+    elif fmt in [CogFrameFormat.ARGB,
+                 CogFrameFormat.xRGB,
+                 CogFrameFormat.ABGR,
+                 CogFrameFormat.xBGR]:
+        return [
+            as_strided(data[1:],
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*4)).transpose(),
+            as_strided(data[2:],
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*4)).transpose(),
+            as_strided(data[3:],
+                       shape=(components[0].height, components[0].width),
+                       strides=(components[0].stride, data.itemsize*4)).transpose()]
+    elif fmt == CogFrameFormat.v210:
+        # v210 is barely supported. Convert it to something else to actually use it!
```

Comment:
As you just return an empty array here, maybe better to raise a NotImplemented exception instead? This seems like un-expected behaviour.